doc/source/user-guide/configuration.rst

```diff
@@ -140,6 +140,23 @@ which result in the following metadata in your HTML pages:
     <meta property="og:site_name" content="PyAnsys" />
     <meta name="physics" content="Electromagnetics" />
 
+If you want to add multiple physics tags, you can do so by providing a list of tags:
+
+.. code-block:: python
+
+   html_context = {
+       "pyansys_tag": ["Electromagnetics", "Structures", "Fluids"],
```

Comment:
If this must be an iterable, then let's rename it to `pyansys_tags`.


```suggestion
       "pyansys_tags": ["Electromagnetics", "Structures", "Fluids"],
```

Reply:
@jorgepiloto Thank you for review, So you prefer to keep, pyansys_tag and pyansys_tags at same time? because the layout has to change based on that, see https://github.com/ansys/ansys-sphinx-theme/pull/758/files#diff-fe62002009468a01eec84a28b20a06966e119f306c93d39f7aaed250c258ed59